### DISCUSSION TITLE
Eintrag "Favoriten Programm" unsichtbar machen?

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#154] [Q&A]
Ich bin noch nicht draufgekommen, wie man den Eintrag "Favoriten Programm" und eventuell "Darkmode" deaktivieren kann.
Könnt ihr mir da weiterhelfen?

![Favoriten_Programm](https://github.com/gis-eni/webgis-forum/assets/100088983/27222fa7-1b5b-44fb-a2e1-a35bb71988df)

**Comment by jugstalt** [comment]
Zu Favoriten Programm:
hast du in der ``portal.config`` einen Eintrag ``use-favorite-detection``? Und wenn ja, ist dieser auf ``true`` gesetzt?

Zu Darkmode:
ergänze deine  ``custom.js`` um folgende folgende Zeile:
``webgis.usability.allowDarkmode = false;``  bzw. wenn es den Eintrag dort schon gibt, setzt der Wert auf von ``true`` auf ``false``

**Reply by faulgurke** [reply]
Perfekt - Danke!

**Comment by ue-black** [comment]
Zum Favoriten Programm:

> "hast du in der portal.config einen Eintrag use-favorite-detection? Und wenn ja, ist dieser auf true gesetzt?"

Ja das haben wir in jeder Instanz eingetragen:

	<add key="use-favorite-detection" value="false" />					


aber nur bei einer Instanz erscheint das Favoriten Programm in der Liste:
![image](https://github.com/user-attachments/assets/e3969769-04c2-4741-8a12-cfd9f8559dea)

Gibt es sonst noch Einstellungen die darauf Auswirkungen haben?

**Reply by gis-eni** [reply]
Nein, das ist die einzige Einstellung die man setzen muss. Eigentlich muss man das erst einmal auf ``true`` setzten damit der Menüpunkt aufscheint. Also wenn es den Key nicht gibt, sollte da nix kommen. Hast du den Eintrag vielleicht 2x in der portal.config?

PS: Nach dem ändern Application Pool neu gestartet?
Welche Version verwendet ihr?

**Reply by ue-black** [reply]
Version 6.24.1502
Application Pool neu gestartet
Es gibt keinen einziges portal.config das nicht auf "false" gesetzt ist bei favorite-detection.

Noch irgendwelche Ideen?

**Reply by jugstalt** [reply]
Updaten auf die aktuelle Version könnte helfen
